### PR TITLE
fix: Catch errors when starting

### PR DESCRIPTION
**Pull request by hofmeister**

### CODE REVIEW

File: src/instances/routes.ts [hunk 1]
```diff
@@ -41,12 +41,16 @@ router.get('/:systemId/instances/:instanceId', (req: Request, res: Response) =>
...
-    res.status(202).send({
-        ok: true,
-        taskId: task.id,
-    });
+        res.status(202).send({
+            ok: true,
+            taskId: task.id,
+        });
+    } catch (e: any) {
+        res.status(500).send({ ok: false, error: e.message });
```
Comment (wejendorp) [inline src/instances/routes.ts:52]: It should be possible for us to use a general error middleware - esp w/ promise-router, where we should be covered for async errors as well. That way we can have a uniform error response format, without potential copy-pasta errors or missing handlers.

**Review by wejendorp — APPROVED** [review]
👍 but would prefer error middleware on the whole server.

File: src/instances/routes.ts (hunk 1 above)
Comment (hofmeister) [reply src/instances/routes.ts:52]: We already have that - this is to avoid hitting sentry

Comment (wejendorp) [reply src/instances/routes.ts:52]: @hofmeister in that case how about refining the filter in the middleware? Rethrowing these with a property or as an HTTP error type.

**Pull request merged**